python/whylogs/api/writer/whylabs_client.py

```diff
@@ -425,8 +432,20 @@ def tag_custom_performance_column(
             column=column,
             default_metric=default_metric,
         )
+
+        @backoff.on_exception(
+            backoff.expo,
+            ApiException,
+            giveup=lambda e: e.status != 429,  # type: ignore
```

Comment:
Do we not retry on 500?

Reply:
Not at present

Reply:
We should also retry on 5xx, but we can do that in a follow up.